src/olympia/devhub/templates/devhub/versions/list.html

```diff
@@ -128,7 +138,7 @@
             <th>{{ _('Currently on AMO') }}</th>
             <th>{{ _('Status') }}</th>
             <th>{{ _('Validation') }}</th>
-            <th class="version-delete">{{ _('Delete') }}</th>
+            <th class="version-delete">{{ _('Delete/Disable') }}</th>
```

Comment:
Hmm.  Delete is one of the two functions though (even though its soft delete) @jvillalobos ?